packages/gx-agent/src/agent/CredentialHandler.ts

```diff
@@ -82,8 +82,6 @@ export class CredentialHandler {
       },
       // purpose: args.purpose, // todo: Make dynamic based on signInfo and arg
       keyRef: signInfo.keyRef,
-      challenge: args.challenge ? args.challenge : GXComplianceClient.getDateChallenge(),
-      domain: args.targetUrl ?? this.config().complianceServiceUrl,
```

Comment:
Why are you removing the arguments/support for using domain/challenge altogether. Please note that this is just a generic method to create a VP, so that isn't necessarily for the CS. You still need to be able to provide these arguments